musig/src/hash.rs

```diff
@@ -0,0 +1,81 @@
+use bellman::pairing::ff::{PrimeField, PrimeFieldRepr};
+use franklin_crypto::eddsa::PublicKey;
+use franklin_crypto::jubjub::JubjubEngine;
+use franklin_crypto::util::sha256_hash_to_scalar;
+use sha2::{Digest, Sha256};
+
+pub const PACKED_POINT_SIZE: usize = 32;
+
+pub trait AggregateHash<E: JubjubEngine> {
```

Comment:
Fixed

Reply:
BTW because of changing this to what you suggested it's now impossible to change hash type in runtime from wasm. It can now use only one hash and it should be predefined

Reply:
If one will want to provide multiple hashers for the WASM, they'll be able to create one more crate with required predefined hashers. And since the hash set is a somewhat stable topic, I think it's OK.